service-map/src/main/java/fi/nls/oskari/map/layer/group/link/OskariLayerGroupLinkService.java

```diff
@@ -17,5 +17,6 @@
 
     public void replace(OskariLayerGroupLink old, OskariLayerGroupLink link);
     public boolean hasLinks(int groupId);
+    public List<OskariLayerGroupLink> findByMaplayerId(int layerId);
```

Comment:
There's a findByLayerId() in the interface already. What would be the difference between these two methods?